bot/utils/pagination.py

```diff
@@ -115,15 +218,16 @@ def event_check(reaction_: Reaction, user_: Member) -> bool:
                     # Reaction is on this message
                     reaction_.message.id == message.id,
                     # Reaction is one of the pagination emotes
-                    str(reaction_.emoji) in PAGINATION_EMOJI,  # Note: DELETE_EMOJI is a string and not unicode
+                    str(reaction_.emoji) in PAGINATION_EMOJI,
                     # Reaction was not made by the Bot
                     user_.id != ctx.bot.user.id,
```

Comment:
```suggestion
                    user.id == ctx.author.id,
```
Currently it only checks if the reaction is made by the bot or not, if we simply change it to check whether it was made by the author then it would also restrict other users from controlling a person’s command.